packages/terra-docs/src/Placeholder.module.scss

```diff
@@ -22,5 +22,13 @@
     color: var(--terra-docs-placeholder-inner-color, #787575);
     display: flex;
     justify-content: center;
+
+    p {
```

Comment:
use a class instead of styling the p tag

Reply:
Updated here https://github.com/cerner/terra-core/pull/3452/commits/05fb3a469732f8968f1c93a9459db6d2d7ad7b62.